runtimevar/runtimeconfigurator/runtimeconfigurator.go

```diff
@@ -168,7 +168,17 @@ func (o *URLOpener) OpenVariableURL(ctx context.Context, u *url.URL) (*runtimeva
 	}
 	var rn ResourceName
 	rn.ProjectID = u.Host
-	rn.Config, rn.Variable = path.Split(u.Path)
+	path := u.Path
+	if len(path) > 0 && path[0] == '/' {
+		path = path[1:]
+	}
+	if pathParts := strings.Split(path, "/"); len(pathParts) == 2 {
```

Comment:
variable can be hierarchical so we should use `strings.SplitN(path, "/", 2)` instead.

Reply:
Done.